input/docs/handbook/data-binding/xamarin-forms.md

```diff
@@ -1,63 +1,50 @@
-In the app.xaml file register your viewmodels (If you want the viewmodel based navigation)
-```    
-this.Router = new RoutingState();
```

Comment:
We have a separate page for Routing https://reactiveui.net/docs/handbook/routing/ that apparently is empty. Perhaps, we should move that contents there instead of removing completely. 

This pages and the ones for other platforms actually describe how to use `IViewFor` and activation on various platforms and are referenced from Bindings page https://reactiveui.net/docs/handbook/data-binding/

Reply:
I agree.  I think a simple "Want to know how this affects View Model based routing?" type link to the other doc page is the right path.